src/app/api/advocates/route.ts

```diff
@@ -3,10 +3,13 @@ import { advocates } from "../../../db/schema";
 import { advocateData } from "../../../db/seed/advocates";
 
 export async function GET() {
-  // Uncomment this line to use a database
-  // const data = await db.select().from(advocates);
-
-  const data = advocateData;
-
-  return Response.json({ data });
+  try {
+    const data = await db.select().from(advocates);
+    return Response.json({ data }, { status: 200 });
+  } catch (error) {
+    console.error("Failed to fetch advocates:", error);
+    console.error("DB error, falling back to static data:", error);
+    // Fallback to static data
+    return Response.json({ data: advocateData, fallback: true }, { status: 200 });
+  }
```

Comment:
[Backend] Fixes: https://github.com/chrisrcoles/solace/pull/1 No Error Handling for src/app/api/advocates/route.ts

The introduction of the try/catch block now allows for better error handling. It allows the code to handle errors gracefully. Now, when an error occurs, the server returns the mock data instead of failing. This way the app can still function.